app/src/main/java/org/mozilla/focus/widget/BrowserNavigationOverlay.kt

```diff
@@ -107,7 +104,8 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
     override fun onClick(view: View?) {
         var event = NavigationEvent.fromViewClick(view?.id) ?: return
         if (event == NavigationEvent.TURBO) {
-            isTurboEnabled = !turboButton.isActivated
+            turboButton.toggle()
```

Comment:
I'd like to put this into the view's onClick, but I'm not sure how to call the super of the view, either here or earlier when I set the onClick listeners.